pkg/agent/variables.go

```diff
@@ -120,6 +122,8 @@ func getCSECommandVariables(config *datamodel.NodeBootstrappingConfiguration) pa
 		"sgxNode":                         strconv.FormatBool(datamodel.IsSgxEnabledSKU(profile.VMSize)),
 		"configGPUDriverIfNeeded":         config.ConfigGPUDriverIfNeeded,
 		"enableGPUDevicePluginIfNeeded":   config.EnableGPUDevicePluginIfNeeded,
+		"migNode":                         strconv.FormatBool(datamodel.IsMIGNode(config.GPUInstanceProfile)),
+		"gpuInstanceProfile":              config.GPUInstanceProfile,
```

Comment:
is this var used?

Reply:
migNode is used in cse_cmd.sh

Reply:
hmm...don't we pass all the scripts through go templating anyway? is there a reason for adding the var here vs as a go template of some sort inside the script?

genuinely curious, I don't know the pros/cons and usually add to the templates except for artifacts

Reply:
I'd assume both are fine but wanted to check